src/fmu/dataio/dataio.py

```diff
@@ -48,12 +49,22 @@
 AggregatedData: Final = AggregatedData  # Backwards compatibility alias
 InitializeCase: Final = InitializeCase  # Backwards compatibility alias
 
-
 # ======================================================================================
 # Private functions
 # ======================================================================================
 
 
+def _future_warning_preprocessed() -> None:
+    warnings.warn(
+        "Using the ExportData class for re-exporting preprocessed data is no "
+        "longer supported. Use the dedicated ExportPreprocessedData class "
+        "instead. In a deprecation period the ExportPreprocessedData is used "
+        "under the hood when a filepath is input to ExportData. "
+        "This will change in the future!",
```

Comment:
"Please update your script, as this will be discontinued in the future."? (call to action)